accepted/future-releases/class-modifiers/feature-specification.md

```diff
@@ -513,23 +515,16 @@ Many combinations don't make sense:
     are mutually exclusive.
 *   `sealed` types cannot be constructed so it's redundant to combine with
     `abstract`.
-*   `sealed` types cannot be extended or implemented, so it's redundant to
-    combine with `final`, `base`, or `interface`.
-*   `sealed` types cannot be mixed in outside of their library, so it
-    contradicts `mixin` on a class. *It's useful to allow `sealed` on a mixin
-    declaration because the mixin can be applied within the same library.
-    A `sealed mixin class`  does not provide any significant extra
-    functionality over a `sealed mixin`, you can replace `extends MixinClass`
-    with `with Mixin`, so a `sealed mixin class` is not allowed.*
-*   `interface` and `final` classes would prevent a mixin class from being used
-    as a superclass or mixin outside of its library. *Like for `sealed`, an
-    `interface mixin class` and `final mixin class` are not allowed, and
-    `interface mixin` and `final mixin` declaration are recommended instead.*
+*   `sealed` types cannot be mixed in, extended or implemented,
+    so it's redundant to combine with `final`, `base`, or `interface`.
```

Comment:
Indeed, that's better. Edited the suggestion to do that.

(I don't think it's redundant, of course, but I don't expect language team support for making all properties explicit. Actually, it would probably be too noisy, and we can always take another dog fight about whether or not it should be shown on hovering on `sealed` and on each mixin name in an anonymous mixin application ;-)

Reply:
Done. (Including the "already".)